0020-bluetooth-payment-protocol/0020-bluetooth-payment-protocol.md

```diff
@@ -0,0 +1,117 @@
+# Bluetooth Payment Protocol
+
+This document specifies a protocol for communicating and authorizing Interledger Payment Requests over Bluetooth (Low Energy). It is primarily intended for use in in-person retail payment contexts.
+
+This protocol may be implemented by Customer devices such as smartphones or very low-power, offline devices such as tapFi's.
+
+## Definitions
```

Comment:
The ones that are general ILP terms should go in the glossary, but some of the terms used here are specific to this protocol. Unless they're used elsewhere I think they should just be defined here